pulp_2to3_migration/app/plugin/content.py

```diff
@@ -267,6 +282,8 @@ def get_remote_by_importer_id(importer_id):
                                 remote=remote,
                                 deferred_download=not downloaded)
                             d_artifacts.append(da)
+                            lce.is_migrated = True
+                        Pulp2LazyCatalog.objects.bulk_update(objs=lces, fields=['is_migrated'])
```

Comment:
Thinking more about it, it doesn't seem to be safe to flip the flag before RemoteArtifacts are created. Imagine a power cut after you flipped the flag (in the first stage) and before RemoteArtifacts are saved (in the penultimate stage), it would mean that we'll never migrate those on the re-run unless a Remote has changed.
Maybe should add those LCEs to the extra_info and handle it in the custom RemoteArtifactSaver stage or in a new stage which will be responsible for LCEs and which will be after RemoteArtifactSaver. What do you think?

Reply:
Created a stage